kube_downscaler/scaler.py

```diff
@@ -800,23 +822,34 @@ def scale_down(
             "Normal",
             dry_run,
         )
-    resource.annotations[ORIGINAL_REPLICAS_ANNOTATION] = str(replicas)
-
+    if resource.kind == "PodDisruptionBudget" and replicas_is_percentage:
```

Comment:
probably dont need that clarification
```suggestion
    if replicas_is_percentage:
```